src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/ComponentAccessibilityCodeActionProvider.cs

```diff
@@ -236,5 +229,11 @@ private bool IsTagUnknown(MarkupStartTagSyntax startTag, RazorCodeActionContext
             }
             return false;
         }
+
+        private sealed class TagHelperPair
```

Comment:
I know you didn't add this but odd to see a private sealed class.

Reply:
Updated to `private class TagHelperPair`